File: Back/.Net/BetFriends.Domain/Events/BetCreated.cs
using BetFriends.Domain.Bets;
using BetFriends.Domain.Members;
using BetFriends.Shared.Domain;

namespace BetFriends.Domain.Events;

public record BetCreated(BetId BetId, MemberId OwnerId, int Coins) : IDomainEvent;

File: Back/NodeJS/modules/bets/src/domain/bets/events/BetCreated.ts
import { IDomainEvent } from "../../../../../shared/domain/IDomainEvent";
import { MemberId } from "../../members/MemberId";
import { BetId } from "../BetId";

export class BetCreated implements IDomainEvent {
    constructor(public BetId: BetId, public MemberId: MemberId, public Coins: number){}
    Type: string = BetCreated.name;
}